plugins/extractit/icon_extractit/util/extractor.py

```diff
@@ -15,12 +15,23 @@
 def extract(provided_regex: str, provided_string: str, provided_file: str) -> list:
     matches = []
     if provided_string:
-        provided_string = urllib.parse.unquote(provided_string)
-        matches = regex.findall(provided_regex, provided_string)
+        mixedmatches = regex.findall(provided_regex, provided_string)  # regex finds both encoded and unencoded urls
```

Comment:
```suggestion
        mixed_matches = regex.findall(provided_regex, provided_string)  # regex finds both encoded and unencoded urls
```
Python convention is snake case